atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -416,7 +417,13 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
         self.arches = arches or []
         self.multilib_arches = []
         self.multilib_method = None
-        self.modular_tags = data.get('modular_koji_tags', [])
+        self.modular_tags = data.get('modular_koji_tags')
+        self.koji_tag = koji_tag
+
+        if self.modular_tags is True:
```

Comment:
Just `if self.modular_tags:` is enough?

Reply:
From the related osbs-client PR (https://github.com/containerbuildsystem/osbs-client/pull/988) it seems that `modular_koji_tags` may use different types to mean different things.

My question is: what exactly does `True` mean here? I couldn't find details in the related JIRA issue.

Reply:
> My question is: what exactly does `True` mean here? I couldn't find details in the related JIRA issue.


Nevermind, found it:

> we can support modular_koji_tags: true in osbs by simply passing the buildroot tag to ODCS

^ is that still valid / correct? IIUC we are passing the build target tag, not the buildroot tag here.

Reply:
buildroot target in this context means koji buildroot for target, don't have it confused with OUR buildroot :)

Reply:
and yes we support either True or list

in case we have list we will use that as modular_koji_tags

if it is True we will use tag from target in list

Reply:
updated docstr